src/image_builder.py

```diff
@@ -110,6 +110,16 @@ def image_builder(buildspec):
                 labels[var] = file_name
                 labels[f"{var}_URI"] = uri
 
+        if str(BUILDSPEC["framework"]).startswith("huggingface"):
+            if image_config.get("transformers_version"):
+                extra_build_args["TRANSFORMERS_VERSION"] = image_config.get("transformers_version")
+            else:
+                raise Exception(f"HuggingFace buildspec.yml must contain 'transformer_version' field for each image")
```

Comment:
Please make this a specific Exception like `MissingImageBuildParameterException`, or generic like a `KeyError`.